src/main/java/io/stargate/sgv2/jsonapi/service/schema/CollectionManager.java

```diff
@@ -0,0 +1,59 @@
+package io.stargate.sgv2.jsonapi.service.schema;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import io.stargate.bridge.proto.Schema;
+import io.stargate.sgv2.api.common.schema.SchemaManager;
+import io.stargate.sgv2.jsonapi.exception.ErrorCode;
+import io.stargate.sgv2.jsonapi.exception.JsonApiException;
+import io.stargate.sgv2.jsonapi.service.schema.model.JsonapiTableMatcher;
+import java.util.function.Function;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class CollectionManager {
```

Comment:
I think adding this class was not needed.. I can simply use the `SchemaManager` in the ops.. I see no future need for extending this one, thus I might remove it..

Reply:
Done.